changelog.md

```diff
@@ -1,3 +1,75 @@
+<a name="5.6.0"></a>
+# 5.6.0 (2019-10-17)
+[Full Changelog](https://github.com/googlechrome/lighthouse/compare/v5.5.0...v5.6.0)
+
+We expect this release to ship in the DevTools of Chrome 79.
+
+## New contributors!
+
+Thanks to @dpacassi, @housseindjirdeh, @lirantal and @yeonjuan for their first contributions!
```

Comment:
@housseindjirdeh is already in https://github.com/GoogleChrome/lighthouse/releases/tag/v5.0.0